docs/album/search.md

```diff
@@ -0,0 +1,126 @@
+Get the top tags for an album on Last.fm, ordered by popularity.
+
+No authentication required.
+
+## Parameters
+
+| Method    | Type                                                                                              | Default | Required                      | Description                                 |
+| --------- | ------------------------------------------------------------------------------------------------- | ------- | ----------------------------- | ------------------------------------------- |
+| `limit`   | [number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) | 30      | :negative_squared_cross_mark: | The artist which's album should be fetched. |
+| `page`    | [number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) | 1       | :negative_squared_cross_mark: | The album that should be fetched.           |
+| `album`   | [string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) | `none`  | :white_check_mark:            | The album's musicbrainz id.                 |
+| `api_key` | [token](https://www.last.fm/api/account/create)                                                   | `none`  | :white_check_mark:            | A Last.fm API key.                          |
+
+## Responses
+
+Errors:
+
+- 2 : Invalid service - This service does not exist
+- 3 : Invalid Method - No method with that name in this package
+- 4 : Authentication Failed - You do not have permissions to access the service
+- 5 : Invalid format - This service doesn't exist in that format
+- 6 : Invalid parameters - Your request is missing a required parameter
+- 7 : Invalid resource specified

```

Comment:
You can leave error 2, 3, 4, 5 and 7 out, as they don't happen since this method. Those errors would only happen if you spelled the method wrong, so we've decided to keep them out of the errors.